index.js

```diff
@@ -1,7 +1,79 @@
+let countWords = 0;
+for (let i = 0; i < longText.length; i++) {
+    if (longText[i] === " ") {
+        countWords++;
+    }
+}
+console.log(`The number of words in the string is ${countWords + 1}`)
+
+let countEt = 0;
+for (let i = 0; i < longText.length; i++) {
+    if (longText.slice(i, i + 4) === " et ") {
+        countEt++;
+    }
+}
+console.log(`The number of times the Latin word "et" appears is ${countEt}`);
+
+// Bonus 2
+
+const phraseToCheck = "A man, a plan, a canal, Panama!";
+let phraseModified = "";
+for (let i = 0; i < phraseToCheck.length; i++) {
+  if (phraseToCheck[i] !== " " && phraseToCheck[i] !== "," && phraseToCheck[i] !== "?" && phraseToCheck[i] !== "!" && phraseToCheck[i] !== "'") {
+    phraseModified += phraseToCheck[i].toLowerCase();
+  }
+}
+
+let phraseReverse = "";
+for (let i = phraseModified.length - 1; i >= 0; i--) {
+  phraseReverse += phraseModified[i];
+}
+
+if (phraseModified === phraseReverse) {
+  console.log(`${phraseToCheck} ===> It's a palindrome`);
+} else {
+  console.log(`${phraseToCheck} ===> It's not a palindrome`);
+}
```

Comment:
Esta perfect!, solo una cosilla, en el if del primer FOR si existen mas elementos aparte de esos que estan ahi, tendrias que ir a añadir un nuevo elemento ahi, y eso es poco o malamente mantenible, pregunta que te hago existe una estructura de datos donde le puedas ir metiendo elementos uno de tras de otro? y otra pregunta, existe alguna forma de comprobar esos elementos? Si no ves la pregunta que te hago escribeme por slack y lo vemos! :D